cpp/benchmarks/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ target_include_directories(cuspatial_benchmark_common
 function(ConfigureBench CMAKE_BENCH_NAME)
     add_executable(${CMAKE_BENCH_NAME} ${ARGN})
     set_target_properties(${CMAKE_BENCH_NAME}
-        PROPERTIES RUNTIME_OUTPUT_DIRECTORY "$<BUILD_INTERFACE:${CUSPATIAL_BINARY_DIR}/gbenchmarks>"
+        PROPERTIES RUNTIME_OUTPUT_DIRECTORY "$<BUILD_INTERFACE:${CUSPATIAL_BINARY_DIR}/benchmarks>"
```

Comment:
I don't think it's necessary to keep them in separate directories. Considering the naming is very similar (with a single character difference), it may be easily overlooked. Because cuspatial only has Hausdorff benchmark at the moment. I'm a bit wary that it might get neglected as we start populating the `benchmarks` folder with other NVBench. The concern also goes away once we rewrite Hausdorff benchmark with NVBench and removes gbenchmark dependency.